src/ui/storage/persistentstorage.js

```diff
@@ -102,4 +107,11 @@ export default class PersistentStorage {
     }
     return allParams;
   }
+
+  /**
+   * Get a value for a given key in storage
+   */
+  get (query) {
```

Comment:
nit: should this param be renamed? Is it always the query or is it a given key?

Reply:
copied from `ui/com/searchparams.js` but can change to key if that is more useful!